readme.md

```diff
@@ -553,6 +560,7 @@
 - [Homematic](https://github.com/hobbyquaker/awesome-homematic) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
+- [Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Automated (Visual) Regression Testing.
```

Comment:
> The entry in the Awesome list should:
 Include a short description about the project/theme of the list. **It should not describe the list itself.**
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..